mono/mini/mini-llvm.c

```diff
@@ -8261,6 +8261,8 @@ emit_method_inner (EmitContext *ctx)
 
 	mono_llvm_add_func_attr (method, LLVM_ATTR_UW_TABLE);
 
+	mono_llvm_add_func_attr_nv (method, "no-frame-pointer-elim", cfg->disable_omit_fp ? "true" : "false");
```

Comment:
See https://github.com/mono/mono/blob/e4552f54fbdb3dba903fefe338cdc9998de5c8ed/mono/mini/aot-compiler.c#L1147

It would be better to only emit this attribute exactly when `cfg->disable_omit_fp` is true.